helm-chart/amalthea/templates/rbac/_rbac_rules.tpl

```diff
@@ -26,16 +26,22 @@
   # Note that we do not patch/update/delete them ever.
   - apiGroups:
       - ""
-      - apps
-      - networking.k8s.io
     resources:
-      - statefulsets
       - persistentvolumeclaims
       - services
-      - ingresses
```

Comment:
question: The removal of these and adding them as separate entries just means that openshift does not like them combined like this when you have more than 1 group?

Reply:
The combination ends up with trying to create rules that are invalid in the sense that there's no statefulsets in the default API nor in `networking.k8s.io` so you will get an error (at least as normal user) telling you that you are trying to create rules for which you don't have authority to.